common/pulp_docker/common/tarutils.py

```diff
@@ -114,3 +114,24 @@ def get_youngest_children(metadata):
                 pass
 
     return list(image_ids)
+
+
+def get_image_manifest(tarfile_path):
+    """
+    Given a path to a tarfile, which itself is the product of "skopeo copy"
+    this returns the decoded manifest.json file if it exists.
```

Comment:
Yes, skopeo does produces a manifest json file but for V1 images, this file doesn't exist. I agree that the documentation should not say that the tarfile is a product of skopeo copy since it could be a V1 image which is a product of `docker save`